library/std/src/sys/mod.rs

```diff
@@ -76,3 +76,52 @@ cfg_if::cfg_if! {
         pub mod c;
     }
 }
+
+#[cfg(not(test))]
+cfg_if::cfg_if! {
+    if #[cfg(target_os = "android")] {
+        pub use crate::android::log2f32;
+        pub use crate::android::log2f64;
+    } else {
+        #[inline]
+        pub fn log2f32(n: f32) -> f32 {
+            unsafe { crate::intrinsics::log2f32(n) }
+        }
+
+        #[inline]
+        pub fn log2f64(n: f64) -> f64 {
+            unsafe { crate::intrinsics::log2f64(n) }
```

Comment:
Ah sorry, I should have caught this with my eyes. Previously this was stuff like:

```rust
            #[cfg(target_os = "android")]
            return crate::sys::android::log2f64(n);
            #[cfg(not(target_os = "android"))]
            return unsafe { intrinsics::log2f64(n) };
```

So in the android case, since this is starting from `use crate` it still needs to go through `::sys`.